### ISSUE TITLE
Color problem

### ISSUE THREAD

**Issue by Loopercal** [zendesklabs/project_app#44]
Hi,

First of all, thanks for creating a great organizing tool. More than 160 people are using it on a daily basis in my company, and it's helping us more than you think.

We are having an issue with how colors are representing ticket statuses (On Hold-black, Pending- blue etc), as since yesterday they are not displayed, and all tickets are marked with white (screenshot attached) which causes confusion as tickets marked with white are indicating closed/solved status. Please note that linked tickets in ‘Open’ status are still shown correctly. 

![project app](https://user-images.githubusercontent.com/35332998/34823104-7fb7cbda-f6c9-11e7-8958-c792198f5462.PNG)

Problem was confirmed by multiple agents, and application restart did not help. It's also possible this is somehow associated with application framework issues Zendesk had yesterday, which literally stopped all of added apps on our account.

Can you please helps us?

**Comment by skipjac** [comment]
Ok I am also seeing it with pending and on-hold

**Comment by Loopercal** [comment]
Hey Skipjac, does that mean we can hope for a fix from your side any time soon? :)

**Comment by robinschrijer** [comment]
Hi,

As of this week we also have the this issues with the colors of the tickets status. It would be great if the same colors could match the zendesk main application.

**Comment by skipjac** [comment]
I plan on working on it over the weekend

**Comment by Loopercal** [comment]
Hey Skipjac, any news on this? QQ

**Comment by skipjac** [comment]
I have push a fix to the app store over the weekend, waiting on them.

On Jan 16, 2018 05:54, "Loopercal" <notifications@github.com> wrote:

Hey Skipjac, any news on this? QQ

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/zendesklabs/project_app/issues/44#issuecomment-357967166>,
or mute the thread

**Comment by Loopercal** [comment]
Hi Skipjac,

Works great now, thank you very very much. <3
